pandas/tests/dtypes/test_missing.py

```diff
@@ -1,3 +1,4 @@
+from contextlib import nullcontext
```

Comment:
I didnt include the try/except for Python 3.6 (see below), is that ok? I thought we don't need that anymore since the CI is running on at least 3.7
```
try:
    from contextlib import nullcontext
except ImportError:
    from contextlib import ExitStack as nullcontext  # Py 3.6.
```